src/dropboxHelper.js

```diff
@@ -0,0 +1,121 @@
+// Functions for storing notes to dropbox
+import { pathJoin, readBlobAsString } from './utils';
+import { dropboxConfig } from './userConfig';
+import { noteNameFromIdAndTitle } from './NoteData';
+
+const NOTES_FOLDER_PATH = pathJoin(
+    dropboxConfig.notesFolderParent === '/' ? '' : dropboxConfig.notesFolderParent,
+    dropboxConfig.notesFolderName
+);
+
+// Helper class for saving and retrieving notes from dropbox.
+class DropboxHelper {
```

Comment:
A cleaner way would be to have dropbox context. And usually classes are unnecessary. With context, components can require the helper functions where ever needed.

Reply:
OK. Not familiar with context but will be a good time to get familiarized and do that.

Reply:
https://reactjs.org/docs/context.html